site/src/util/formUtils.ts

```diff
@@ -45,6 +46,17 @@ export const getFormHelpers =
     }
   }
 
+export const getFormHelpersWithError = <T>(
+  form: FormikContextType<T>,
+  error?: Error | unknown,
```

Comment:
I'm surprised `error` is optional in this fn.

Reply:
Ha, yeah. This will eventually replace `getFormHelpers`, but we want to be able to replace it piecemeal so we had to give it a different name. It's called `withError` because the functionality it adds is unpacking an error (so that we don't have to repeat that code everywhere). A different name would be fine though!